packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -3307,8 +3308,9 @@ class SemanticsConfiguration {
   /// This is a request to decrease the value represented by the widget. For
   /// example, this action might be recognized by a slider control.
   ///
-  /// If a [value] is set, [decreasedValue] must also be provided and
-  /// [onDecrease] must ensure that [value] will be set to [decreasedValue].
+  /// If [this.value] is set, [decreasedValue] must also be provided and
```

Comment:
What will be printed in the HTML docs if we use `[this.value]`? Will it literally say `this.value`, or will it just say `value`? Because just considering the output prose, it's pretty clear that "value" refers to "this.value" to the reader, so it's kind of redundant to say it that way.